### PR TITLE
sriov pkg: added GetPciAddess and GetDriverName functions

### PR DESCRIPTION
**Pull request by evgenLevin**

### CODE REVIEW

File: pkg/sriov/networknodestate.go [hunk 1]
```diff
@@ -169,16 +204,16 @@ func (builder *NetworkNodeStateBuilder) GetNumVFs(sriovInterfaceName string) (in
...
 
 	for _, interf := range builder.Objects.Status.Interfaces {
 		if interf.Name == sriovInterfaceName {
-			return interf.NumVfs, nil
+			return &interf, nil
 		}
 	}
 
-	return 0, fmt.Errorf("failed to find interface %s", sriovInterfaceName)
+	return nil, fmt.Errorf("interface %s not found", sriovInterfaceName)
```
Comment (achuzhoy) [inline pkg/sriov/networknodestate.go:216]: nit: is or was not found

Comment (evgenLevin) [reply pkg/sriov/networknodestate.go]: fixed

**Comment by kononovn** [comment]
@evgenLevin Please re-name PR headline and commit message to:
sriov pkg: added GetPciAddess and GetDriverName functions

**Comment by evgenLevin** [comment]
@kononovn renamed

**Review by kononovn — APPROVED** [review]
/lgtm

**Review by achuzhoy — APPROVED** [review]

**Pull request merged**